src/js_sync.hpp

```diff
@@ -833,19 +833,26 @@ void SyncClass<T>::populate_sync_config(ContextType ctx, ObjectType realm_constr
             ssl_verify_callback = std::move(ssl_verify_functor);
         }
 
-        bool is_partial = false;
+        bool is_partial = true;
+        ValueType full_synchronization_value = Object::get_property(ctx, sync_config_object, "full_synchronization");
         ValueType partial_value = Object::get_property(ctx, sync_config_object, "partial");
+
         if (!Value::is_undefined(ctx, partial_value)) {
             is_partial = Value::validated_to_boolean(ctx, partial_value);
         }
 
-        bool disable_partial_sync_url_checks = false;
-        ValueType disable_partial_sync_url_checks_value = Object::get_property(ctx, sync_config_object, "_disablePartialSyncUrlChecks");
-        if (!Value::is_undefined(ctx, disable_partial_sync_url_checks_value)) {
-            disable_partial_sync_url_checks = Value::validated_to_boolean(ctx, disable_partial_sync_url_checks_value);
+        // In the deprecation period setting full_synchronization takes precedence over partial
```

Comment:
I would rather throw an exception.

Reply:
I agree if both `partial` and `full_synchronization` is sat by the user - that should throw.

Reply:
You mean throw an exception if both `partial` and `fullSynchronization` is set? That works for me as well